lib/src/auth/auth_storage.dart

```diff
@@ -19,8 +19,20 @@ class AuthStorage implements IAuthTokenStorage {
   }
 
   @override
-  Future<void> storeAuthToken(String token) async {
-    return await _storage.write(key: BedrockService.tokenKey, value: token);
+  Future<void> storeAuthToken(String refreshToken) async {
+    return await _storage.write(
+        key: BedrockService.refreshToken, value: refreshToken);
```

Comment:
I just tried testing login flow and it's currently broken due to the refresh token. I am not sure if Bedrock back-end currently supports refreshing tokens but it might be something to look into.

Reply:
Sorry that was a typo, shouldn't had been renamed to `refreshToken`.